apps/web/src/hooks/useRequiredParams.ts

```diff
@@ -70,6 +70,7 @@ const useRequiredParams = <
   nonEmptyParams = keyArray.reduce((reducedParams, key) => {
     const param: string | undefined = params[String(key)];
     let newParams = { ...reducedParams };
+    console.log(isUUID(param ?? ""));
```

Comment:
Can this be removed?

Reply:
Absolutely! [remove console log statement](https://github.com/GCTC-NTGC/gc-digital-talent/pull/8815/commits/2aa36b77fe1ac60ac2364a58257e5f0e3bec5711)